### PR TITLE
feat: :tada: init

### LINKED ISSUE #1: UI for creating a form kit schema
I as a user want to be able to configure a formkit schema (https://formkit.com/essentials/schema) by adding elements.

I want to be able to add elements of the following types
- text https://formkit.com/inputs/text
- textarea https://formkit.com/inputs/textarea
- number https://formkit.com/inputs/number

For every element I need to be able to specify the following attributes (which align with the formkit element props):
- name
- label
- help

While configuring the schema I want to see a live preview of the configured form.

The ouput of this whole thing should be the formkit schema, so that I can copy it to somewhere else.

### PR DESCRIPTION
**Pull request by lukashass**
Closes #1

### CODE REVIEW

**Comment by mariusheine** [comment]
@anbraten @lukashass how is the status of this PR? Can you finish this by today so that it is published and usable by tomorrow?

**Comment by mariusheine** [comment]
@anbraten Ok and do you have time to finish it by tomorrow 12PM?

File: packages/former/src/components/FormContent.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <FormKit
+      type="button"
+      :label="mode === 'preview' ? 'Edit' : 'Preview'"
+      :onClick="() => (mode = mode === 'edit' ? 'preview' : 'edit')"
+    />
+    <div ref="el">
+      <FormKitSchema :schema :library />
+    </div>
+    <div class="mt-4 mx-auto">
+      <FormKit v-if="schema.length === 1" type="button" label="Add component" :onClick="addComponent" />
```
Comment (mariusheine) [inline packages/former/src/components/FormContent.vue:12]: @anbraten Why this `v-if` With this implementation you can only add one component?

File: packages/former/src/components/FormKitEdit.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      </div>
+
+      <div class="ml-auto gap-4 hidden group-hover:flex">
+        <FormKit type="button" label="Edit" :onClick="edit" />
+        <FormKit type="button" label="Delete" :onClick="onDelete" />
+      </div>
+    </div>
+
+    <div class="mx-auto mt-2 hidden group-hover:flex">
+      <FormKit type="button" label="Add component" :onClick="addAfter" />
```
Comment (mariusheine) [inline packages/former/src/components/FormKitEdit.vue:15]: @anbraten And all these buttons are not visible. Is this related to the not working tailwind config? Seems to be the case for me...

File: packages/former/src/components/FormContent.vue (hunk 1 above)
Comment (anbraten) [reply packages/former/src/components/FormContent.vue:12]: That's intended as the add button is part of each entry later on to be able to add elements between two existing ones.

File: packages/former/src/components/FormKitEdit.vue (hunk 2 above)
Comment (anbraten) [reply packages/former/src/components/FormKitEdit.vue:15]: It's part of the tailwind issue.
